test/jasmine/tests/command_test.js

```diff
@@ -508,6 +513,14 @@ describe('component bindings', function() {
         }).catch(fail).then(done);
     });
 
+    it('udpates bound components when the value changes', function(done) {
+        expect(gd.layout.sliders[0].active).toBe(0);
+
+        Plotly.restyle(gd, 'marker.color', 'ecru').then(function() {
```

Comment:
Ah, of course. My mistake. I added `black` which is valid but not present. I also modified the test name to read `does not update the component if the value is not present`.